db/schema.rb

```diff
@@ -64,12 +64,6 @@
             name: "index_contacts_on_contactable"
   end
 
-  create_table "data_migrations",
-               primary_key: "version",
-               id: :string,
-               force: :cascade do |t|
-  end
-
```

Comment:
This gets auto generated for the `rails data:migrate`, I am assuming it's not there if you haven't run local data migrations.  Please run rails data:migrate to put this back

Reply:
@taw27 weird - I did a DB dump to make sure it was in line and this got removed.. What version of PG are you on?

Reply:
nvm I misread that, will do

Reply:
@LaurelOlson data migrations are separate from db:migrations and don't get run automatically (intentionally). If you run rails data migrate or rails db:migrate:with_data this table will get generated along with the data migrations (which isn't relevant for clean db)

Reply:
OK fixed, thanks!